package.json

```diff
@@ -112,13 +112,13 @@
     "vsce:package": "vsce package",
     "onprem:prep": "yarn onprem:prep:package && yarn onprem:prep:readme && yarn onprem:prep:cloudHost",
     "onprem:prep:package": "yarn onprem:prep:backuppackage && yarn onprem:prep:includeBinaries && npm pkg set \"name=tabnine-vscode-enterprise\" && yarn onprem:prep:webpack && npm pkg delete __metadata",
-    "onprem:prep:cloudHost": "cp package.json package.json.back && cat package.json | jq '.contributes.configuration[0].properties += {\"tabnine.cloudHost\": {\"type\": \"string\",\"default\": \"https://tabnine.sec.samsung.net\",\"description\": \"The tabnine cloud host to connect to\"}}' > package.json.onprem && cp package.json.onprem package.json",
+    "onprem:prep:cloudHost": "cp package.json package.json.back && cat package.json | jq '.contributes.configuration[0].properties += {\"tabnine.cloudHost\": {\"type\": \"string\",\"default\": \"https://tabnine.sec.samsung.net\",\"description\": \"The tabnine cloud host to connect to\"}}' > package.json.onprem1 ; cat package.json.onprem1 | jq '.contributes.configuration[0].properties += {\"tabnine.department\": {\"type\": \"string\",\"enum\": [\"department1\", \"department2\", \"department3\"], \"description\": \"The department you belong to\", \"default\": \"departement2\"}}' > package.json.onprem && cp package.json.onprem package.json",
```

Comment:
@yanir-codota  provide the list of departments here: (replace with what is needed)

```
[\"department1\", \"department2\", \"department3\"] <<< replace with the correct list

\"default\": \"departement2\" <<< consider adding a "Undefined" case
```